### PR TITLE
fix: packages dependencies & animted rename 

### PR DESCRIPTION
**Pull request by MonsterAnan**
Hi, I very like this project.Now a lot of people are using react-native-reanimted v2.
Upgrading to react-native-reanimted v2 is highly recommended. 😄
So:
- Project's react-native-reanimted version should depend on myself project.
- fix: "WARN  Easing was renamed to EasingNode in Reanimated 2. Please use EasingNode instead"

So this PR is for v2, thanks!!!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,8 +37,5 @@
     "eslint": "^7.4.0",
     "eslint-plugin-react": "^7.20.3"
   },
-  "dependencies": {
-    "react-native-reanimated": "^1.10.2",
-    "react-native-gesture-handler": "^1.7.0"
-  }
+  "dependencies": {}
```
Comment (heyman333) [inline package.json:40]:
```suggestion
  "peerDependencies": {
      "react-native-reanimated": "^2",
      "react-native-gesture-handler": "^2"
  }
```

**Comment by heyman333** [comment]
Thank you for your work! @MonsterAnan 
I just commented on some line!
please check it!

**Comment by MonsterAnan** [comment]
Yea,you are right!
Thank you for you reply!

**Comment by heyman333** [comment]
@MonsterAnan 
Oh I was just giving an example, I thought it would be nice

```js
"peerDependencies": {
      "react-native-reanimated": ">=2",,
  }
```

could you fix it?

**Comment by MonsterAnan** [comment]
Oh,sorry, I edited it on my phone the other night, I thought you fixed it 🤣.
Now It's been fixed, I think there is no problem. 🤝 @heyman333

**Comment by MonsterAnan** [comment]
Hi, could you publish npm?
My project need it.
Thanks.

**Comment by heyman333** [comment]
@MonsterAnan 
hey. I just published npm. Please let me know if you have ploblem.

**Comment by MonsterAnan** [comment]
thank you! 💗
I have no more problems.

**Pull request merged**